docs/sdk/dev/development-tools/performance-audit-tool-benchmark.md

```diff
@@ -150,12 +150,12 @@ Before you start writing the tests, check out the [PHPBench documentation](https
 
 You can write tests without any additional knowledge, but Benchmark has some default helpers that can make writing benchmarks easier for Spryker:
 
-* **RequestBuilder** - helps you to build the `Request` object
-* **HttpHelper** - sends a request
-* **LoginHelper** - allows you to log in with some credentials during or before the benchmark
-* **FormCsrfTokenHelper** - allows you to get valid CSRF token if you want to submit some form on the page
+* **RequestBuilder** - helps you to build the `Request` object.
+* **HttpHelper** - sends a request.
+* **LoginHelper** - allows you to log in with some credentials during or before the benchmark.
```

Comment:
use "let" instead of "allow" https://developers.google.com/style/word-list#allows-you-to
for em-dashes, don't use spaces
```suggestion
* **LoginHelper**—lets you log in with some credentials during or before the benchmark.
```